edx_reverification_block/xblock/reverification_block.py

```diff
@@ -100,26 +75,30 @@ def student_view(self, context=None):
             related_assessment=related_assessment
         )
 
-        if verification_status:
-            # TODO: What message will be displayed to user if it is already has any status?
-            fragment.add_content(unicode(verification_status))
+        has_skipped_verification = self.runtime.service(self, "reverification").is_user_skipped_reverification(
+            user_id, course_id
+        )
+        if has_skipped_verification:
+            fragment.add_content(unicode("You have skipped re-verification"))
         else:
-            reverification_link = self.runtime.service(self, "reverification").start_verification(
-                course_id=course_id,
-                related_assessment=related_assessment,
-                item_id=item_id
-            )
-            org = self.get_org()
-            html = self._render_template(
-                "static/html/reverification.html",
-                {
-                    'reverification_link': reverification_link,
-                    'org': org
-                }
-            )
-            fragment.add_content(html)
-            fragment.add_javascript(self._resource("static/js/skip_reverification.js"))
-            fragment.initialize_js('SkipReverifcation')
+            if verification_status:
+                # TODO: What message will be displayed to user if it is already has any status?
+                fragment.add_content(unicode(verification_status))
```

Comment:
nitpick: `# TODO: What message will be displayed to user if it is already has a status?`